src/preamble.js

```diff
@@ -1055,8 +1060,16 @@ function createWasm(env) {
 #endif
   }
 
+#if 'emscripten_generate_pc' in addedLibraryItems
+  addRunDependency('offset-converter');
```

Comment:
Is there a need for adding run dependencies here? looks like the offset converter is just plain code, so there is no async startup we need to handle for it, unless I'm missing something? (we should only add run deps if they are necessary for proper startup)

Reply:
Well, when we use the `fetch` API in the async loading case, we need to do `response.clone().arrayBuffer()` to get the WASM binary to feed into the offset converter. However, `response.clone().arrayBuffer()` returns a promise, which is why I added a run dependency.